backend/tests/test_upload_reservation_routes.py

```
@@ -62,12 +62,12 @@ def test_successful_upload(client, init_db):
         "document_type": "ID",
         "document_number": "ABC123456",
         "cf": "JHNDOE90A01X123Y",
-        "frontImage": (
+        "frontimage": (
             open("tests/test_images/front.jpeg", 'rb'),
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)